creator-node/src/diskManager.js

```diff
@@ -113,7 +122,10 @@ class DiskManager {
    */
   static extractCIDsFromFSPath (fsPath) {
     const match = CID_DIRECTORY_REGEX.exec(fsPath)
-    if (!match || !match.groups) return null
+    if (!match || !match.groups) {
+      genericLogger.info(`Path is not a directory, fsPath=${fsPath}`)
```

Comment:
I think if the regex doesn't match with `fsPath`, that the input doesnt follow the pattern `file_storage/Qm.../Qm...` or `file_storage/Qm...`. can we update this msg to reflect that?

Reply:
I don't think we check that closely, but also debugging regex always takes me time:

https://github.com/AudiusProject/audius-protocol/blob/44d578021bb83365dbcba91020bc5b834838df40/creator-node/src/diskManager.js#L7-L10

I mainly copied the comments that were included with `CID_DIRECTORY_REGEX` that were both set during the same PR.

Thoughts?

Reply:
perhaps something like `Input path does not match cid directory pattern, fsPath=${fsPath}`?